config/user_config_windows_test.go

```diff
@@ -1,9 +1,62 @@
 package config
 
-import "testing"
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUserConfig(t *testing.T) {
+	dir, err := ioutil.TempDir("", "user-config")
+	assert.NoError(t, err)
+	defer os.RemoveAll(dir)
+
+	cfg := &UserConfig{
+		Config: New(dir, "user"),
+	}
+	cfg.Token = "a"
+	cfg.Workspace = "/a"
+
+	// write it
+	err = cfg.Write()
+	assert.NoError(t, err)
+
+	// reload it
+	cfg = &UserConfig{
+		Config: New(dir, "user"),
+	}
+	err = cfg.Load(viper.New())
+	assert.NoError(t, err)
+	assert.Equal(t, "a", cfg.Token)
+	assert.Equal(t, filepath.Join(cfg.Home, "a"), cfg.Workspace)
+}
 
 func TestNormalizeWorkspace(t *testing.T) {
-	// I don't have access to a windows machine,
-	// so I am guaranteed to get this wrong.
-	t.Fatal("Need to implement normalization tests for windows.")
+	cwd, err := os.Getwd()
+	assert.NoError(t, err)
+
+	cfg := &UserConfig{Home: "C:\\Users\\alice"}
+	tests := []struct {
+		in, out string
+	}{
+		{"", ""}, // don't make wild guesses
+		{"C:\\Users\\alice\\\\\\foobar", "C:\\Users\\alice\\foobar"},
+		{"/foobar", "C:\\Users\\alice\\foobar"},
```

Comment:
An extra test from the Darwin ones for assuming that `/foobar` is in the home directory.